File: ccppath-sdk/cs/tests/ClientTest.cs
using System;
using Aliyun.SDK.CCP.CCPClient;
using Aliyun.SDK.CCP.CCPClient.Models;
using Xunit;

namespace tests
{
    public class ClientTest
    {
        [Fact]
        public void Test_ListDrives()
        {
            Config config = new Config();
            config.DomainId = "sz16";
            config.Protocol = "http";
            config.AccessKeyId = Environment.GetEnvironmentVariable("ACCESS_KEY_ID");
            config.AccessKeySecret = Environment.GetEnvironmentVariable("ACCESS_KEY_SECRET");
            Client client = new Client(config);
            ListDriveRequest request = new ListDriveRequest();
            RuntimeOptions runtime = new RuntimeOptions();
            ListDriveResponse result = client.ListDrives(request, runtime);
            Assert.Equal(2, result.Items.Count);
            Assert.Equal("", result.NextMarker);
        }
    }
}

File: ccppath-sdk/java/src/test/java/com/aliyun/ccp/ccpclient/ClientTest.java
package com.aliyun.ccp.ccpclient;

import com.aliyun.ccp.ccpclient.models.*;
import org.junit.Before;
import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class ClientTest {

    private static Client client;

    @Before
    public void createClient() throws Exception {
        Config config = new Config();
        config.domainId = "sz16";
        config.protocol = "http";
        config.accessKeyId = System.getenv("RAMAccessKeyId");
        config.accessKeySecret = System.getenv("RAMAccessKeySecret");
        client = new Client(config);
    }

    @Test
    public void listDrives() throws Exception {
        ListDriveRequestModel requestModel = new ListDriveRequestModel();
        ListDriveRequest request = new ListDriveRequest();
        requestModel.body = request;
        RuntimeOptions runtime = new RuntimeOptions();
        ListDrivesModel result = client.listDrives(requestModel, runtime);
        assertEquals(2, result.body.items.size());
        assertEquals("", result.body.nextMarker);
    }
}
